cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,63 @@
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
+  const user = {
+    firstName: ["John", "Konor", "Will", "Illia", "Wolter"],
+    lastNames: ["Smith", "Wick", "Danirus", "Brown", "White", "Miller"],
+    subjects: ["Math", "Science", "English", "History", "Art"],    
+    address: ["Khreshchatyk", "Golden", "Poshaina"]
+  }
+
+  const randomFirstName = user.firstName[Math.floor(Math.random() * user.firstName.length)];
+  const randomLastName = user.lastNames[Math.floor(Math.random() * user.lastNames.length)];
+  const randomIndex = Math.floor(Math.random() * 3);
+  const randomHobby = Math.floor(Math.random() * 3) + 1;
+  const randomSubject = user.subjects[Math.floor(Math.random() * user.subjects.length)];
+  const randomAddress = `${user.address[Math.floor(Math.random() * user.address.length)]} 69`
+
+  const email = `${randomFirstName.toLowerCase()}${randomLastName.toLowerCase()}@test.com`
+  const phoneNumber ="0" + Math.floor(Math.random() * 1000000000).toString().padStart(9, "0");
+
   before(() => {
-    
+    cy.visit('/');
   });
 
-  it('', () => {
+  it('should allow to register new user', () => {
+    cy.viewport(550, 750);
+
+    cy.findByPlaceholder('First Name')
+      .type(randomFirstName);
+
+    cy.findByPlaceholder('Last Name')
+     .type(randomLastName);
+
+    cy.get('#userEmail').type(email);
+    
+    const genderList = cy.get("#genterWrapper > .col-md-9 > :nth-child(1), #genterWrapper > .col-md-9 > :nth-child(2), #genterWrapper > .col-md-9 > :nth-child(3)");
+    genderList.eq(randomIndex).click();
+
+    cy.findByPlaceholder('Mobile Number').type(phoneNumber);
```

Comment:
it's hard to read your test because of the mix of the test and variables. please move your test data to a separate file